cachito/web/api_v1.py

```diff
@@ -29,8 +29,19 @@
 from cachito.web.utils import deep_sort_icm, pagination_metadata, str_to_bool
 from cachito.workers import tasks
 
+from prometheus_client import multiprocess
+from prometheus_client.core import CollectorRegistry
+from prometheus_flask_exporter import PrometheusMetrics
+
+
 api_v1 = flask.Blueprint("api_v1", __name__)
 
+# Prometheus Flask Exporter.
+#   - gets us our metrics.
+registry = CollectorRegistry()
```

Comment:
Why do we need this initialization both in `cachito/web/api_v1.py` and `cachito/web/api_v1.py`?

Reply:
Aha, I see you deleted it in the second commit. Why it was needed?

Reply:
It had been moved in order to use decorators on the API definitions.   Turns out the decorators weren't needed to achieve the ability to group the stats by endpoint, and not broken out by request ID.  It would be best to keep it here if in the future anything needs to be done to customize the metrics gathered for an endpoint.

Reply:
Also squashed.